### DISCUSSION TITLE
atproto/api: Support fetching multiple users feeds within one request (Feature Request)

### DISCUSSION THREAD

**Discussion by GoldenAngel2** [bluesky-social/atproto#1900] [Bluesky Lexicons]
**Is your feature request related to a problem? Please describe.**
The main issue is not being able to fetch multiple users feed(s) within one request.

- Why is this such a big issue? 

Currently In order to get multiple users feed(s) you have to do the requests one-by-one that cause more stress on the site/api host(s) in the long run. 

**Describe the solution you'd like**
Add something like `<BskyAgent>.getAuthorFeeds({ actors: [ "string", "string2" ] })` and return the multiple users feeds etc.
- Have a max limit on how many "actors" that can be fetched per-request (`1000`) or have it customizable per-host 🤷

**Describe alternatives you've considered**
Fetching the users one by one which wouldn't be great for the host 

**Additional context**
N/A

**Comment by bnewbold** [comment]
Is the idea to fetch something like a unified feed with content from all the users, or multiple distinct feeds?

For the former: you could add the individual accounts to a List, and then fetch the list feed.

For the later: feed responses are already fairly large and resource-intensive to fetch for a single user. We do generally want the API to be usable for devs, and are open to adding new batched requests to reduce round-trips. It isn't obvious to me that this would be a good candidate for that pattern though. The overall latency of the request would be high, and the response body quite large.
